rfc/2024-12-12-combinining-authenticator-and-account-store.md

```diff
@@ -0,0 +1,109 @@
+```
+Title: Bring account store and authenticator into the combined pre-award monolith
+Owner: Marc Usher
+Collaborator(s): Steven Fountain, Gideon Goldberg, Sam Williams
+Created on: 2024-12-04
+Status: Draft
```

Comment:
I guess status is Approved for this one?

Reply:
pft no, why would it be 🫣